save-sandbox/src/test/kotlin/com/saveourtool/save/orchestrator/controller/agents/AgentsControllerTest.kt

```diff
@@ -62,6 +63,10 @@ class AgentsControllerTest {
         )
         whenever(dockerService.createContainers(any(), any()))
             .thenReturn(listOf("test-agent-id-1", "test-agent-id-2"))
+
+        whenever(agentRunner.getContainerIdentifier(any())).thenReturn("save-test-agent-id-1")
```

Comment:
twice `whenever(agentRunner.getContainerIdentifier(any()))`, does it work?

Reply:
It did somehow
Changed to one invocation